### PR TITLE
fix window.onmessage event proxing (handler has 'object' type) (close #133)

### LINKED ISSUE #133: Wrong proxing window.onmessage event handler if it has 'object' type
Page for reproduce:
```javascript
<html>
	<body>
		<h1 id="msg"></h1>
		<br/>
		<input type="button" value="Send message" id="sendMsg">
		<script>
		    var eventHandler = {
				handleEvent: function(e) {
					var msg = document.getElementById('msg');
					msg.textContent = e.data.toString();
				}
			};
			window.addEventListener('message', eventHandler);
			
			var sendMsgButton = document.getElementById('sendMsg');
			sendMsgButton.addEventListener('click', function() {
				window.postMessage('test message reseived', '*');
			});
		</script>
	</body>
</html>
```

Open page in playground and click 'Send message' button.
In console we've got a script error:
'Uncaught TypeError: originListener.call is not a function'

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @inikulin

### CODE REVIEW

File: test/client/fixtures/sandbox/message-test.js [hunk 1]
```diff
@@ -14,6 +14,21 @@ QUnit.testDone(function () {
     iframeSandbox.off(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, initIFrameTestHandler);
 });
 
+asyncTest('onmessage event (handler has \'object\' type (GH-133))', function () {
```
Comment (inikulin) [inline test/client/fixtures/sandbox/message-test.js]: You can get rid of escaping here and brackets shouldn't be nested:
```js
'onmessage event (handler has "object" type) (GH-133)'
```

**Comment by inikulin** [comment]
Just a minor style issue
\r-

**Comment by miherlosev** [comment]
FPR

File: test/client/fixtures/sandbox/message-test.js [hunk 2]
```diff
@@ -14,6 +14,21 @@ QUnit.testDone(function () {
     iframeSandbox.off(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, initIFrameTestHandler);
 });
 
+asyncTest('onmessage event (handler has "object" type (GH-133))', function () {
```
Comment (inikulin) [inline test/client/fixtures/sandbox/message-test.js]: No nested brackets!

**Comment by inikulin** [comment]
\r-

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by miherlosev** [comment]
ping @churkin

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
Stop, commit misses `closes` directive.
\r-

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

File: test/client/fixtures/sandbox/message-test.js [hunk 3]
```diff
@@ -14,6 +14,24 @@ QUnit.testDone(function () {
...
 
+asyncTest('onmessage event (handler has "object" type) (GH-133)', function () {
+    var testMessage       = 'test';
+    var testPropertyValue = 'propertyValue';
+
+    var eventHandlerObject = {
+        propertyName: testPropertyValue,
+        handleEvent:  function (e) {
+            strictEqual(e.data, testMessage);
+            strictEqual(this.propertyName, testPropertyValue);
```
Comment (churkin) [inline test/client/fixtures/sandbox/message-test.js]: `strictEqual(this.propertyName, testPropertyValue);`

replace to:

 `strictEqual(this, eventHandlerObject);`

**Comment by churkin** [comment]
lgtm

**Pull request merged**